tests/network-tests/src/flows/membership/updatingProfile.ts

```diff
@@ -18,15 +18,28 @@ export default async function updatingProfile({ api, query }: FlowProps): Promis
   const updates: MemberProfileData[] = [
     // Partial updates
     // FIXME: Currently handle always need to be provided, see: https://github.com/Joystream/joystream/issues/2503
-    { handle: 'New handle 1', name: 'New name' },
-    { handle: 'New handle 2' },
+    {
+      handle: 'New handle 1',
+      name: 'New name',
+      validatorAccount: 'validator address',
+    },
+    {
+      handle: 'New handle 2',
+      validatorAccount: 'validator address2',
```

Comment:
I know I suggested 'validator address2' before but just to make sure it works with `undefined`:
```suggestion
```
It would be a more useful test IMO.

Reply:
the fourth update is `undefined`

Reply:
Yes but before the fourth update `validatorAccount` has been set to `undefined` already (during the 3rd one) so I think having `undefined` on the 2de update is more valuable